tomcat10/src/main/java/com/linecorp/armeria/server/tomcat/TomcatService.java

```diff
@@ -445,7 +445,7 @@ public final HttpResponse serve(ServiceRequestContext ctx, HttpRequest req) thro
                 });
             } catch (Throwable t) {
                 logger.warn("{} Failed to invoke Tomcat:", ctx, t);
-                res.close();
+                res.abort(t);
```

Comment:
- Since the exception is propagated, we may remove the warning message.
  ```java
  res.abort(new IllegalStateException("Failed to invoke Tomcat", t);
  ```
- Should we also abort `res` when an exception is raised in L440 ~ L443?

Reply:
> Should we also abort res when an exception is raised in L440 ~ L443?

Updated, but I didn't add a test case since I couldn't think of a good way to verify this change 😅